package.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/package.json",
   "name": "opentok",
-  "version": "2.17.1",
+  "version": "2.18.1",
```

Comment:
This needs to be reverted. We should not be adjusting version numbers in PRs but only when we deploy. It causes a lot of merge issues otherwise

Reply:
@manchuck, I'm not familiar with your process. So, we should merge this in with the version unmodified, then what:

* Modify the version locally and deploy it?
* Add another PR to rev the version?

Reply:
@jeffswartz After publishing, we push up the new version. We can bypass branch protection for that since its just a version bump. It looks like the commit for v18 was not pushed up before I went on paternity leave (my bad sorry about that). Once this PR merges in and we publish v2.18.1 (which is what this one should be) then all the commits will be present in the repo. 

The process for publishing the version is:

1. pull down latest code 
1. run `npm version <major|minor|patch>`
1. run `npm publish`
1. Force push up the commit (some of us have permission to do this for this very reason)

That is why we do not need to have version numbers bumped in PRs. We can have a case where I open a PR which has version `2.18.0` as a change while this one is open. Depending on which version PR closes first, the other PR will not be able to merge in due to conflicts